src/page/PeoplePage.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage:React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoader, setLoader] = useState(true);
+  const [isError, setError] = useState(true);
+  const [messageError, setMessageError] = useState('');
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    setError(true);
+    getPeople()
+      .then(response => {
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code